### PR TITLE
Adds ntd ridership numbers and impact analysis

### PR DESCRIPTION
**Pull request by vevetron**
- Added a query that pulls ridership numbers
- Adjusts LA metro bus and rail to get close to matching their distributions
- Removes duplicate ntd entries
- Creates graph and tables with estimated ridership impacts.

### CODE REVIEW

File: holiday_service_research/holiday_research.ipynb [hunk 1]
```diff
@@ -230,6 +245,177 @@
...
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "upt = query_sql(f\"\"\"\n",
+    "SELECT year, ntd_id as ntd_id_2022, agency_name, reporter_type, time_period, sum(unlinked_passenger_trips__upt_) as sum_unlinked_passenger_trips_upt  \n",
+    "FROM `cal-itp-data-infra.mart_ntd.dim_annual_ntd_agency_service` \n",
+    "where ntd_id in ({ntd_ids_for_query})\n",
+    "and time_period = 'Annual Total'\n",
+    "and year = 2022\n",
```
Comment (evansiroky) [inline holiday_service_research/holiday_research.ipynb:400]: Does this query need to be modified to filter out inactive and demand-response ridership?

Comment (vevetron) [reply holiday_service_research/holiday_research.ipynb:401]: It doesn't need to filter out inactive ridership.

I lumped in the demand-response ridership with the regular fixed-route ridership.  I assumed it's more likely the holiday service would affect demand-response as well so any impact of bad schedules would affect both regular fixed-route and demand-responose the same.

Comment (evansiroky) [reply holiday_service_research/holiday_research.ipynb:401]: ah, ok. Seems like a reasonable assumption.

**Review by evansiroky — APPROVED** [review]
Looks great. Thanks for all of this work!

**Pull request merged**